db_test.go

```diff
@@ -2275,19 +2277,67 @@ func TestWithContextQueryRewriter(t *testing.T) {
 	// Test db.QueryRow
 	logger.lastQuery = ""
 
-	db.QueryRow("SELECT * from objects LIMIT 1")
+	db.QueryRow(q)
 
-	if logger.lastQuery != "/* test */ SELECT * from objects LIMIT 1" {
-		t.Fatalf("Expected %q, got %q", "/* test */ SELECT * from objects LIMIT 1", logger.lastQuery)
+	if logger.lastQuery != "/* test */ "+q {
+		t.Fatalf("Expected %q, got %q", "/* test */ "+q, logger.lastQuery)
 	}
 
 	// Test with white space
 	logger.lastQuery = ""
 
 	db.QueryRow(`
-     SELECT * from objects LIMIT 1`)
+     ` + q)
+
+	if logger.lastQuery != "/* test */ "+q {
```

Comment:
This can be neater if you define `expected := "/* test */ "+q` etc. for each test case